src/frontend/screens/LeaveProject/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { useState } from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import {
+  NavigationStackProp,
+  NavigationStackScreenComponent,
+  NavigationStackScreenProps,
+} from "react-navigation-stack";
+import { LeaveProjectCompleted } from "./LeaveProjectCompleted";
+import { LeaveProjectInitial } from "./LeaveProjectInitial";
+import { LeaveProjectProgress } from "./LeaveProjectProgess";
+import HeaderTitle from "../../sharedComponents/HeaderTitle";
+
+const m = defineMessages({
+  headerTitle: {
+    id: "screens.LeaveProject.LeaveProject",
+    defaultMessage: "Leave Project",
+  },
+});
+
+enum ScreenStates {
+  initial,
+  progress,
+  completed,
+}
+
+export interface ILeaveSharedProp {
```

Comment:
Not a big deal but i'd lean towards making this more descriptive with the name because the `I` doesn't really give me much context as a newcomer

Reply:
I think `I` is a pretty standard way for defining interfaces in the ts world! But we can also develop a best practice for us if you feel like that would be better!

Reply:
ah got it. yeah i thought it was an abbreviation for `Initial` 😂 i've seen that pattern but usually for things that are closer to generics or for library authors, so i think i'd personally lean away from that for app code